docs/blog/2018-08-30-gatsby-values/index.md

```diff
@@ -0,0 +1,167 @@
+
+**By taking the time up front to think through our goals and plan appropriately, we will avoid costly mistakes and slowdowns, keep the team and community on the same page, and ultimately move _much faster_** than if we were charging ahead without thinking things through.
+
+### Always prioritize developer experience and user experience
+
+Whenever we make decisions, we will put the experience of the developers using Gatsby and the people who will use the sites generated by Gatsby first. Even if it means extra work to add helpful error messages. Even if it means refactoring a confusing API to be more user friendly.
+
+**Gatsby has built its reputation on being a pleasure to work with, and we’re dedicated to keeping that true.**
+
+### Embrace growth
+
+**Healthy things grow, and growing things change.** Gatsby is a fast-growing, cutting edge technology startup, and that means things are in near-constant flux. This means that we’re regularly up against circumstances that are rapidly evolving.
+
+In order to survive and grow, we need to embrace growth as a healthy, vital process. This means making pragmatic decisions early, delegating when necessary, asking for help when it’s time, and recognizing how we’ve always done things isn’t necessarily what works _today_. (And what works today may very well stop working tomorrow.)
+
+These changes aren’t a sign that we’re failing, or that things are wrong. It’s a sign that the company is thriving.
+
+### Experiment with cool ideas
+
+Gatsby began as an experiment with bleeding edge technology. As a product, Gatsby has always aimed to be at the forefront of technological advances.
+
+While we need to stick to our roadmap and make steady progress toward our company goals, **we will maintain a state of excitement about new technology and foster a culture of experimentation, play, and wonder.** We’re excited about reaching our goals and love to tinker. People are constantly creating incredible things, and it’s critical both to our success as a company _and_ to our happiness as geeks that we take time to play with these incredible new technologies.
+
+### You belong here
+
+Open source doesn’t have the best reputation for being friendly and welcoming, and that makes us sad. **Everyone belongs in open source, and Gatsby is dedicated to making you feel welcome.**
+
+We will never judge, condescend, or exclude anyone. Instead, we will go above and beyond to support the community, through [pair programming](https://gatsby.app/pairing), offering [free swag for contributors](https://gatsby.app/swag), giving control to the community by [auto-inviting all contributors to the Gatsby GitHub org](https://github.com/gatsbyjs/gatsby/pull/7699#issuecomment-416665803), or any other means that empowers and embraces the incredible community that makes Gatsby possible.
```

Comment:
Does it make sense to link to code of conduct somewhere here? Because the code of conduct is meant to create an inclusive and welcoming environment, which means that any behaviors that make others feel unwelcome are not tolerated, and a list of the behaviors we will not tolerate and how we manage them are defined in Code of Conduct.

Reply:
Yeah, great idea 👍

Reply:
Excellent idea. I'll add that now.